staging/publishing/rules.yaml

```diff
@@ -1991,10 +1991,6 @@ rules:
       dir: staging/src/k8s.io/pod-security-admission
   library: true
 recursive-delete-patterns:
-- BUILD
-- '*/BUILD'
-- BUILD.bazel
-- '*/BUILD.bazel'
 - Gopkg.toml
```

Comment:
@nikhita: can we delete this now as well? I can't imagine we've had a Gopkg.toml for ... ages now 🤔

Reply:
hehe yep

Reply:
dep shouldn't be used by anyone anymore as well filed https://github.com/kubernetes/kubernetes/pull/112870